src/libsync/filesystem.cpp

```diff
@@ -45,6 +46,16 @@ extern "C" {
 
 namespace OCC {
 
+QString FileSystem::longWinPath( const QString& inpath )
+{
+    QString path(inpath);
+
+    path.replace('/', '\\');
+    path.prepend(QLatin1String("\\\\?\\"));
```

Comment:
So this is the same as makeLongWinPath but in 3 lines of code? Could you possibly not have two implementations somehow?